src/org/zowe/jenkins_shared_library/pipelines/Build.groovy

```diff
@@ -193,13 +193,13 @@ class Build {
 
     private Integer identifyCause(Cause c) {
         def causeID
-        if (c instanceof jenkins.branch.BranchEventCause) {
+        if (c.toString().contains("BranchEventCause")) {
```

Comment:
This line is just to make jenkins_library build pass, because it somehow cannot find this library.

Reply:
Not sure if you printed out, do you know what's the new value of `c.toString()`?

Reply:
I printed out the c.properties, one of the key value pair is "class":"class jenkins.branch.BranchIndexEvent"
Since c is already a Cause object, its inherited toString(), according to the docs, is getting: 
getClass().getName() + '@' + Integer.toHexString(hashCode())